networkx/algorithms/tree/tests/test_distance_measures.py

```diff
@@ -26,6 +26,24 @@ def test_tree_centroid_multiple_centroids(self):
         G = nx.full_rary_tree(2, 8)
         assert nx.tree_centroid(G) == [0, 1]
 
+    def test_tree_centroid_different_from_graph_center(self):
+        G = nx.Graph(
+            [
+                (0, 1),
+                (0, 2),
+                (0, 3),
+                (0, 4),
+                (0, 5),
+                (0, 6),
+                (6, 7),
+                (7, 8),
+                (8, 9),
+                (9, 10),
+            ]
+        )
```

Comment:
I think this is clearer, but feel free to disregard.
```suggestion
        G = nx.Graph()
        nx.add_star(G, [0, 1, 2, 3, 4, 5, 6])
        nx.add_path(G, [6, 7, 8, 9, 10])
```

Reply:
lgtm